File: Spring Boot/monolith/src/main/java/pl/edu/pg/benchmarking/monolith/distance/Point.java
package pl.edu.pg.benchmarking.monolith.distance;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor(access = AccessLevel.PUBLIC)
public class Point {

    private Double latitude;

    private Double longitude;
}

File: Dotnet/monolith/Models/Point.cs
namespace Monolith.Models
{
    public class Point
    {
        public double Latitude { get; set; }
        public double Longitude { get; set; }

        public Point()
        {

        }

        public Point(double lat, double lon)
            => (Latitude, Longitude) = (lat, lon);
    }
}
